### PR TITLE
Godeps,vendor: bump ofnet to b44a11b

### PR DESCRIPTION
**Pull request by unclejack**
This PR bumps ofnet to switch to contiv/libOpenflow. This new libOpenflow library includes fixes for goroutines leak and race conditions. ofnet hasn't been modified in any other way. The only changes I've made to ofnet itself were the ones required to accommodate the new import paths.

The libOpenflow code has been properly tested and benchmarked. The contiv/libOpenflow PR which made these changes has also added some benchmarks and tests.

### CODE REVIEW

File: vendor/github.com/contiv/libOpenflow/util/stream.go [hunk 1]
```diff
@@ -35,6 +37,8 @@ type MessageStream struct {
 	pool *BufferPool
 	// Message parser
 	parser Parser
+	// Channel to shut down the parser goroutine
+	parserShutdown chan bool
```
Comment (rchirakk) [inline vendor/github.com/contiv/libOpenflow/util/stream.go]: when do we send parser shutdown event ?

Comment (dseevr) [inline vendor/github.com/contiv/libOpenflow/util/stream.go]: It's sent later on in the diff of this file:

https://github.com/contiv/libOpenflow/commit/d9c950917b660d7a8258ffe5842f863e065f7d2f#diff-16ba25332f097a22ef695fe81b87c6e7R90

Comment (rchirakk) [inline vendor/github.com/contiv/libOpenflow/util/stream.go]: >> parser shutdown event 
how is it  triggered from netplugin & how to verify/test ?

Comment (dseevr) [inline vendor/github.com/contiv/libOpenflow/util/stream.go]: Looking at the original PR/discussion might give more context:  https://github.com/contiv/libOpenflow/pull/1

The tests are also in that PR

Comment (unclejack) [inline vendor/github.com/contiv/libOpenflow/util/stream.go]: @rchirakk The comments on that PR should show exactly where that happens. The outbound message processing goroutine has a select which watches the shutdown channel, in addition to watching for outbound messages to be sent out. We already had the means to shut down that outbound message processing queue. That PR has added a for loop which tells the individual goroutines which process inbound messages to shut down - just as was done before for the outbound messages goroutine. This means that the main shutdown procedure of this message sending & receiving code triggers the shutdown of the inbound message processing goroutines the same way the outbound message queue was shut down before.

This has no impact on performance or on how the code works in the end. If that code had been broken or its behavior had been altered in any way which caused issues, I would've discovered that.

**Comment by unclejack** [comment]
I've added some more comments on the original PR. It seemed to be a good idea to explain those changes in order to make it clear they're isolated to cleanup and avoiding issues related to the Xid.

**Comment by unclejack** [comment]
Can we merge this?

**Comment by dseevr** [comment]
All of the changes have been approved here:  https://github.com/contiv/libOpenflow/pull/1

Merging per @rhim

**Pull request merged**